WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -615,17 +615,29 @@ extension WordPressAppDelegate {
 extension WordPressAppDelegate {
 
     var currentlySelectedScreen: String {
+        guard let rootViewController = window?.rootViewController else {
+            return String()
+        }
+
         // Check if the post editor or login view is up
-        let rootViewController = window?.rootViewController
-        if let presentedViewController = rootViewController?.presentedViewController {
+        if let presentedViewController = rootViewController.presentedViewController {
```

Comment:
This introduces slightly different logic, but would it clean things up to switch on `rootViewController.presentedViewController ?? rootViewController`?

Reply:
That's a great suggestion, @twstokes ! Thank you. This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/19677/commits/805032b7f6de604f8cb51f27065599b03c701897.